modules/placement-driver/README.md

```diff
@@ -0,0 +1,13 @@
+# Placement driver
+
+The module responses for location of leaseholder node in the replication group. The placement driver decides a node which can hold a lease 
```

Comment:
I'd rephrase this a bit. Let's mention that, on the one hand, placement driver is responsible for primary replica selection and fail-over and on the other hand placement driver provides an API in order to retrieve the information about available primary replicas along with corresponding meta such as leaseholder interval.